### DISCUSSION TITLE
Can't Install?

### DISCUSSION THREAD

**Discussion by DanielTheUncoolio** [elementary/triage#208] [Q&A]
I am installing Elementary OS onto my laptop.
I've installed it before to my Gaming Laptop and I now want to install it to my work laptop.
Specs:
Intel Celeron N4020 (4 cores 10th Gen)
4GB DDR4
128GB SSD
Internet Enabled

When I'm installing, I ALWAYS get an error "Could not Install... Possibly due to a hardware error"

**Comment by DanielTheUncoolio** [comment]
![20230402_122207](https://user-images.githubusercontent.com/117919540/229350002-9dc6375d-ade1-4f9a-9b64-9e5bebc2e34a.JPG)

This is the picture I was able to get of the error.

**Comment by lenemter** [comment]
How are you installing it? Manual partitioning or full disk install?

**Reply by DanielTheUncoolio** [reply]
I manually partitioned it... 
But it didn't work
I then tried the full disk
But it didn't work
But then it did ???
I said the installer failed with the exact same error at the exact same place...
but unlike the other attempts, it just worked after I restarted the computer.

**Comment by 1280px** [comment]
IIRC this has to deal with nasty installer bug where it fails to partition the drive on full-disk installation. Try partitioning the disk yourself (use Custom Install).

**Reply by DanielTheUncoolio** [reply]
I was already.
I tried installing Ubuntu also and it also didnt work... but Ubuntu told me why.
It couldn't install Grub (for some odd reason)
I had to completely delete my Windows partition because it was somehow stopping Grub from installing at all.

**Reply by ghost** [reply]
That may be it. I had windows too! Maybe bwcause of secure boot?

**Reply by DanielTheUncoolio** [reply]
I tried with Secure Boot on and off..
both gave the same result

**Comment by YaYaTurre** [comment]
I am having the exact same problem. Had to switch to another Linux distro, but I would really like to use Elementary os.

**Reply by DanielTheUncoolio** [reply]
I couldn't even switch to another distro.
all of them just kept failing when trying to install Grub.

**Reply by RogerWeihrauch** [reply]
hi
Maybe only the 'Windows OS' option was set in BIOS boot options?
Try to set to any kind of 'Other OS'.
Regards,
Roger

**Comment by ghost** [comment]
Strangely, I had the same error when installing TO a USB flash drive. Installing to my SSD seemed to solve the issue.

**Reply by DanielTheUncoolio** [reply]
I wish I knew how to install TO a USB
Installing to SSD was causing the issues

**Comment by tantanh** [comment]
> Could not Install
> ...due to a hardware error...

I'm facing the same issue. Searching the Internet for solutions but none are working. Please help.

**Reply by djpapeles** [reply]
I'm having the same problem, for me the issue is solved by previously formatting the entired disk in ext4, then the automatic installer run without problems,  this is for a elementary only install not for a dualboot setup.

**Reply by tantanh** [reply]
noted thanks @djpapeles unfortunately I do dualboot with Windows.

**Reply by RogerWeihrauch** [reply]
hi
maybe my answer above is worth to checkout: 'Windows OS' in Bios' boot options.
Regards
Roger

**Comment by sawal123** [comment]
I have the same problem, I'm very confused

**Reply by RogerWeihrauch** [reply]
hi may be also Secure Boot is enabled?
Than, there is much more to do on this.
Pls also see:
https://github.com/orgs/elementary/discussions/275
Regards
Roger

**Comment by mrschneid3r** [comment]
Hey, a bit late to the party, just wanted to share in case someone still needs it. Ok, so I spent 2 days trying to figure out what the f* was wrong with my set up. I've tried everything, I have a 'support' laptop that is an old Acer Aspire, it is a pain in the butt to try to install anything other than windows. What I did:

1. Download the ISO again.
2. Reinstall Etcher.
3. In my case I was burning the ISO on my Mac and using a dongle (obviously), I've had to switch it.
4. Redo the process.

NOW TO THE HOST PHASE

1. Before entering installation phase, go to the built-in terminal if your distro has one.
2. Run the 'lsblk' tool to identify your disk.
3. Then go to 'gdisk'. As soon as you start it might tell you that you have a corrupter GPT and/or MBR Partition Table, send 'z' to erase, and it will dissolve the corrupted partition table.
4. Exit and you are good to go.

P.S. Sometimes you'd have to partition the disk yourself (I do recommend). Also if after you install it doesn't show up on boot, go to your BIOS setting and add the boot file manually (I had to do it).

Good luck.
